datalake/src/main/java/com/sequenceiq/datalake/flow/delete/SdxDeleteActions.java

```diff
@@ -169,14 +169,24 @@ protected void doExecute(SdxContext context, SdxDeletionFailedEvent payload, Map
                 if (exception.getMessage() != null) {
                     statusReason = exception.getMessage();
                 }
-                try {
-                    SdxCluster sdxCluster = sdxStatusService.setStatusForDatalakeAndNotify(DatalakeStatusEnum.DELETE_FAILED, statusReason,
-                            payload.getResourceId());
-                    metricService.incrementMetricCounter(MetricType.SDX_DELETION_FAILED, sdxCluster);
-                } catch (NotFoundException notFoundException) {
-                    LOGGER.info("Can not set status to SDX_DELETION_FAILED because data lake was not found");
+                if (payload.isForced()) {
```

Comment:
I think it is not a good idea to check "forced" flag here. It would be a better approach if we handle it separately in stack deletion and rds deletion phase. With this implementation, it can hide many error which is not related to stack or rds delete. For example there is a nullpointer exception somewhere, then it will set sdx to delete status but it should not happen. We can move a logic like this into StackDeletionHandler and RdsDeletionHandler.